configapi/api_slice_mgmt.go

```diff
@@ -168,11 +169,12 @@ func NetworkSliceDeleteHandler(c *gin.Context) bool {
 	return true
 }
 
-func NetworkSlicePostHandler(c *gin.Context, msgOp int) bool {
+func NetworkSlicePostHandler(c *gin.Context, msgOp int) (bool, string) {
 	sliceName, _ := c.Params.Get("slice-name")
 	if !isValidName(sliceName) {
-		logger.ConfigLog.Errorf("invalid Network Slice name %s. Name needs to match the following regular expression: %s", sliceName, NAME_PATTERN)
-		return false
+		errorMsg := fmt.Sprintf("invalid Network Slice name %s. Name needs to match the following regular expression: %s", sliceName, NAME_PATTERN)
+		logger.ConfigLog.Errorf(errorMsg)
+		return false, errorMsg
```

Comment:
Following my suggestion to change the signature of the function, this code would change in a way similar to this:

```suggestion
		err := fmt.Errorf("invalid Network Slice name %s. Name needs to match the following regular expression: %s", sliceName, NAME_PATTERN)
		logger.ConfigLog.Error(err.Error())
		return err
```

At the end of the function, we would then explicitly return nil.